### PR TITLE
feat: PBI-26 edit team profile

### PR DESCRIPTION
**Pull request by HugoIr**
### Describe your changes
Add endpoint for edit team profile

### Backlog name
[PBI-26 edit team profile](https://monapi.atlassian.net/browse/MON-101?atlOrigin=eyJpIjoiMjBlY2RkMGJhM2VhNDgwYjlmNzhiOWJkZjY4MmY2MmEiLCJwIjoiaiJ9)

### Acceptance criteria
- User can edit team description and team logo
- Team name cannot be changed
- Previously saved image, when changed, is deleted

### Example Request
Request
```
curl --location --request PUT 'http://localhost:8000/team-management/' \
--header 'Authorization: Token xxx' \
--header 'Content-Type: application/json' \
--data-raw '{
    "description": "123",
    "logo": "picture.png"
}'
```
Response status 200 OK
```
{
    "id": xx,
    "name": "test team"
}
```

### Checklist
- [x] I have performed a self-review of my code
- [x] Code successfully run on local
- [x] Feature / changes tested on local
- [x] No failing unit test
- [ ] Passed UAT Test
- [x] Sonarqube tested 
- [ ] Required any changes to environment variable

### CODE REVIEW

**Review by Exqrch — APPROVED** [review]
Sudah single responsibility principle, keren hug

File: team_management/views.py [hunk 1]
```diff
@@ -31,3 +32,24 @@ def create(self, request, *args, **kwargs):
 			return Response(data=serialized_obj.data, status=status.HTTP_201_CREATED)
 			
 		return Response(data={"error": "Please make sure your [team name] is exist!"}, status=status.HTTP_400_BAD_REQUEST)
+
+	def update(self, request, *args, **kwargs):
+		team = Team.objects.get(pk=kwargs['pk'])
```
Comment (andrew-susanto) [inline team_management/views.py:37]: coba ini make sure request.auth.team == team yg mau di update hug

**Comment by Exqrch** [comment]
Sorry, nitip conflict hugo

**Review by ferdifdi — APPROVED** [review]

File: team_management/views.py (hunk 1 above)
Comment (HugoIr) [reply team_management/views.py:39]: donee ya, bolee dicek lagii @andrew-susanto

File: team_management/views.py [hunk 2]
```diff
@@ -34,6 +35,28 @@ def create(self, request, *args, **kwargs):
...
+
+			if (request.data.get('description') is not None):
+				team.description = request.data.get('description')
+			
+			team.save()
+
+			serialized_obj = TeamSerializers(team)
+			return Response(data=serialized_obj.data, status=status.HTTP_200_OK)
+		else:
+			return Response(data={"error": "Require to change team first!"}, status=status.HTTP_400_BAD_REQUEST)
```
Comment (andrew-susanto) [inline team_management/views.py:57]: error messagenya "Team not found" aja hug wkwkk

Comment (HugoIr) [reply team_management/views.py]: wkkwk oke done yaa dru bisa dicek lagi @andrew-susanto

**Review by andrew-susanto — APPROVED** [review]

File: team_management/views.py (hunk 2 above)
Comment (andrew-susanto) [reply team_management/views.py]: approved 🙏

**Pull request merged**